lessons/intro/numpy/index.ipynb

```diff
@@ -3705,7 +3705,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Případně můžu využít možností Jupyter Notebooku a HTML, a zvuk si přehrát:"
+    "Případně můžu využít možností Jupyter Notebooku a HTML a zvuk si přehrát:"
```

Comment:
There is nothing like Oxford comma in Czech language. There is "slučovací poměr" between these two parts => no comma. See: http://prirucka.ujc.cas.cz/?id=153